src/models/__tests__/profile.test.ts

```diff
@@ -184,147 +187,176 @@ describe("Profile codec", () => {
   });
 });
 
-/*
 describe("createProfile", () => {
-  it("should create a new profile", async () => {
-    const clientMock: any = {
-      createDocument: jest.fn((_, newDocument, __, cb) => {
-        cb(undefined, {
-          ...newDocument,
-          _self: "self",
-          _ts: 123
-        });
-      })
-    };
+  const newProfile: NewProfile = {
+    kind: "INewProfile",
+    ...aRawProfile,
+    servicePreferencesSettings: {
+      mode: ServicesPreferencesModeEnum.LEGACY,
+      version: -1
+    }
+  };
 
-    const model = new ProfileModel(clientMock, profilesCollectionUrl);
+  it("should create a new profile", async () => {
+    const containerMock = ({
+      items: {
+        create: jest.fn().mockReturnValue(
+          Promise.resolve({
+            resource: aRetrievedProfile
+          })
+        )
+      }
+    } as unknown) as Container;
 
-    const newProfile: Profile = {
-      fiscalCode: aFiscalCode
-    };
+    const model = new ProfileModel(containerMock);
 
-    const result = await model.create(newProfile, newProfile.fiscalCode);
+    const result = await model.create(newProfile)();
 
-    expect(clientMock.createDocument).toHaveBeenCalledTimes(1);
-    expect(clientMock.createDocument.mock.calls[0][1].kind).toBeUndefined();
-    expect(clientMock.createDocument.mock.calls[0][2]).toHaveProperty(
-      "partitionKey",
-      aFiscalCode
+    expect(containerMock.items.create).toHaveBeenCalledTimes(1);
+    expect(containerMock.items.create).toHaveBeenCalledWith(
+      expect.objectContaining({
+        kind: newProfile.kind,
+        fiscalCode: newProfile.fiscalCode
+      }),
```

Comment:
check that we are calling create with the right parameters. Check even `id`, `version`

Reply:
done in https://github.com/pagopa/io-functions-commons/pull/295/commits/7d00e44bc545338b56405a860cef3bb0c12f8659